src/components/Pages/Acknowledgements.js

```diff
@@ -13,4 +13,6 @@ const Acknowledgements = () => (
   </div>
 );
 
-export default Acknowledgements; 
\ No newline at end of file
+export default Acknowledgements; 
```

Comment:
Are the changes in these JS files accidentally included?

Reply:
Hi. No, they're not by accident. Those changes were one of tasks of my previous "Acknowledgements" page ticket. 

The problem seems to be, that I'm still a newbie, re: PRs. And I noticed this issue while creating the new PR--i.e., the presence of the previous file changes. 

But I didn't know how to push--or create a PR--just for the new change (creation of utilities folder, etc.). Meaning, the previous changes were also re-pushed. But @RNR1 already approved those changes via an earlier PR. 

Sorry for the wordy response, @vontell But does it all (👆) make sense?